### PR TITLE
Started API Integration

### PR DESCRIPTION
**Pull request by SLKhadeeja**
it gives me a 200 but the output is null

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -1,8 +1,14 @@
...
-    news = search.search("jhu", "Data")
-    if news:
-        print(json.dumps(news, indent=4))
+app = FastAPI()
+
+
+@app.get("/")
+async def root():
+    if __name__ == "__main__":
+        news = search.search("jhu", "Data")
```
Comment (MeNsaaH) [inline app.py]: Can you verify that the news is a valid

Comment (SLKhadeeja) [reply app.py]: When I print it normally, it gives an output

Comment (SLKhadeeja) [reply app.py]: > When I print it normally, it gives an output

I also checked type(news) and it's dict

Comment (MeNsaaH) [reply app.py]: Can you remove the if statement, just return the news object

**Comment by SLKhadeeja** [comment]
Turns out `if __name__ == "__main__"` was culprit

**Comment by MeNsaaH** [comment]
Yeah, you only use `if __name__ == __main__` from your code entry point, not in a library

**Comment by MeNsaaH** [comment]
Move the api implementation to a new file `api.py` in the module and create another PR

**Pull request merged**